admin_manual/configuration_server/reverse_proxy_configuration.rst

```diff
@@ -50,6 +107,175 @@ the IP addresses of the proxy. This is useful when you use a reverse SSL proxy
+    location ~ ^/(?:\.|autotest|occ|issue|indie|db_|console) {
+       deny all;
+     }
+
+    location ~ ^/(?:index|remote|public|cron|core/ajax/update|status|ocs/v[12]|updater/.+|ocs-provider/.+|core/templates/40[34])\.php(?:$|/) {
+       include fastcgi_params;
+       fastcgi_split_path_info ^(.+\.php)(/.*)$;
+       fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+       fastcgi_param PATH_INFO $fastcgi_path_info;
+       #Avoid sending the security headers twice
+       fastcgi_param modHeadersAvailable true;
+       fastcgi_param front_controller_active true;
+       fastcgi_pass unix:/run/php/php7.2-fpm.sock;
+       fastcgi_intercept_errors on;
+       fastcgi_request_buffering off;
+    }
+
+    location ~ ^/(?:updater|ocs-provider)(?:$|/) {
+       try_files $uri/ =404;
+       index index.php;
+    }
+
+    # Adding the cache control header for js and css files
+    # Make sure it is BELOW the PHP block
+    location ~* \.(?:css|js)$ {
+        try_files $uri /index.php$uri$is_args$args;
+        add_header Cache-Control "public, max-age=7200";
+        # Add headers to serve security related headers (It is intended to
+        # have those duplicated to the ones above)
+        add_header X-Content-Type-Options nosniff;
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-XSS-Protection "1; mode=block";
+        add_header X-Robots-Tag none;
+        add_header X-Download-Options noopen;
+        add_header X-Permitted-Cross-Domain-Policies none;
+        # Optional: Don't log access to assets
+        access_log off;
+   }
+
+   location ~* \.(?:svg|gif|png|html|ttf|woff|ico|jpg|jpeg)$ {
+        try_files $uri /index.php$uri$is_args$args;
+        # Optional: Don't log access to other assets
+        access_log off;
+   }
+ }
+
+For the config.php portion we have used the following configuration:
+
+::
+
+ <?php
+  $CONFIG = array (
+  'instanceid' => '[IGNORE_THIS_PART]',
+  'passwordsalt' => '[IGNORE_THIS_PART]',
+  'secret' => '[IGNORE_THIS_PART]',
+  'trusted_domains' =>
+  array (
+    0 => 'nextcloud.domain.com',
+  ),
+  'trusted_proxies'   => ['IP_OF_REVERSE_PROXY'],
+  'overwrite.cli.url' => 'https://nextcloud.domain.com',
+  'overwritehost'     => 'nextcloud.domain',
+  'overwriteprotocol' => 'https',
+  'datadirectory' => '/usr/share/nginx/nextcloud/data',
+  'dbtype' => 'mysql',
+  'version' => '13.0.0.14',
+  'dbname' => 'cloud',
+  'dbhost' => 'localhost',
+  'dbport' => '',
+  'dbtableprefix' => 'oc_',
+  'mysql.utf8mb4' => true,
+  'dbuser' => 'oc_superoffice',
+  'dbpassword' => '[IGNORE_THIS_PART]',
+  'installed' => true,
+  'mail_smtpmode' => 'smtp',
+  'mail_smtpauthtype' => 'LOGIN',
+  'mail_smtpsecure' => 'ssl',
+  'mail_from_address' => 'nextcloud',
+  'mail_domain' => 'domain.com',
+  'mail_smtpauth' => 1,
+  'mail_smtphost' => 'smtp.domain.com',
+  'mail_smtpport' => '465',
+  'mail_smtpname' => 'nextcloud@domain.com',
+  'mail_smtppassword' => [IGNORE_THIS_PART]',
```

Comment:
Please don't paste the complete config here. Just the essential parts. So this whole example is not needed, but the single settings bellow are fine.

Reply:
Will make edits